mon-pix/app/controllers/assessments/checkpoint.js

```diff
@@ -15,7 +15,7 @@ export default Controller.extend({
   }),
 
   completionPercentage: computed('finalCheckpoint', 'model.progression.completionPercentage', function() {
-    return this.finalCheckpoint ? 100 : this.model.progression.completionPercentage;
+    return this.finalCheckpoint ? 100 : this.get('model.progression.completionPercentage');
```

Comment:
`this.model.get('progression.completionPercentage')` doit marcher.

Reply:
Les tests ne passent plus dans ce cas là

Reply:
Y'a une raison qui fait qu'on a mis en place le pattern `this.get('model.prop.prop)` vs `this.get('model').prop.prop ? Hormis les `Cannot read property 'prop' of undefined` éventuels ? S'en protéger est parfois un smell d'ailleurs